app/models/manageiq/providers/amazon/cloud_manager/event_parser.rb

```diff
@@ -1,48 +1,41 @@
 module ManageIQ::Providers::Amazon::CloudManager::EventParser
-  # these have their own EmsEvent classes in automate
-  INSTANCE_EVENTS = %w(
-    AWS_EC2_Instance_CREATE
-    AWS_EC2_Instance_UPDATE
-    AWS_EC2_Instance_DELETE
-    AWS_EC2_Instance_running
-    AWS_EC2_Instance_shutting-down
-    AWS_EC2_Instance_stopped
-  ).to_set.freeze
-
-  def self.map_event_type(aws_event_type)
-    if INSTANCE_EVENTS.include?(aws_event_type)
-      aws_event_type
-    else
-      'ConfigurationItemChangeNotification'
-    end
+  def self.parse_config_event!(event, event_hash)
+    event_hash[:message]                   = event["configurationItemDiff"]
+    event_hash[:timestamp]                 = event["notificationCreationTime"]
+    event_hash[:vm_ems_ref]                = parse_vm_ref(event)
+    event_hash[:availability_zone_ems_ref] = parse_availability_zone_ref(event)
   end
 
-  def self.format_message(event)
-    changed_properties = event.fetch_path("configurationItemDiff", "changedProperties")
-    if changed_properties.present?
-      "#{event['eventType']}: #{changed_properties}"
-    else
-      "#{event['eventType']}"
-    end
+  def self.parse_cloud_watch_api_event!(event, event_hash)
+    event_hash[:message]                   = "request: #{event.fetch_path("detail", "requestParameters")}, "\
+                                             "response: #{event.fetch_path("detail", "responseElements")}"
+    event_hash[:timestamp]                 = event["time"]
+    # TODO(lsmola) so Event can be tied to more Vms, we will need to change the modeling
+    event_hash[:vm_ems_ref]                = event.fetch_path("detail", "responseElements", "instancesSet", "items").
```

Comment:
Should there be a similar setter for EBS `volume_ems_ref` or is it sufficient to just store the event?